sky.md

```diff
@@ -42,6 +42,9 @@ On the other hand, a bot might _actually_ be unresponsive, and the scores awarde
 :   When a bot is unable to make a move it is said to be "defeated". Engine inspects the `game-state` at each turn to kill bots that have been "defeated".
 Engine appends a game-summary to the `logs` (when `append_logs`) is called.
 
+**init_syscall_filter()**
```

Comment:
yes we can call it inside **init_bot()** but it would be good if we define in a separate module.